src/web/View.tsx

```diff
@@ -66,21 +67,33 @@ if (typeof document !== 'undefined') {
 
 export interface ViewContext {
     isRxParentAText?: boolean;
+    focusManager?: FocusManager;
 }
 
 export class View extends ViewBase<Types.ViewProps, {}> {
     static contextTypes: React.ValidationMap<any> = {
-        isRxParentAText: PropTypes.bool
+        isRxParentAText: PropTypes.bool,
+        focusManager: PropTypes.object
     };
     context: ViewContext;
 
     static childContextTypes: React.ValidationMap<any> = {
-        isRxParentAText: PropTypes.bool.isRequired
+        isRxParentAText: PropTypes.bool.isRequired,
+        focusManager: PropTypes.object
     };
 
+    private focusManager: FocusManager;
+
     private resizeDetectorAnimationFrame: number;
```

Comment:
These should start with underscores as well. (I didn't catch that in the previous review.)